scripts/kafka_base_dir.sh

```diff
@@ -0,0 +1,6 @@
+if [ "$TRAVIS" == "true" ]
```

Comment:
I prefer `[[` over `[` and I will also prepend the file with the shebang `#!/bin/bash`